ddtrace/contrib/aws_lambda/patch.py

```diff
@@ -11,8 +11,27 @@
 from ddtrace.internal.wrapping import unwrap
 from ddtrace.internal.wrapping import wrap
 
+from .cold_start import is_cold_start
+from .cold_start import set_cold_start
 
-log = get_logger(__name__)
+
+class DDLambdaLogger:
```

Comment:
If this is not a class users would have to interface with, we should avoid making it part of the public interface. Similarly, the `cold_start` imports above will be in the public interface for this package, and so should be avoided. It might make better sense to move all this code to an internal `_cold_start.py` file.